lib/src/components/NxTab/NxTab.scss

```diff
@@ -6,16 +6,51 @@
  */
  @import '../../scss-shared/nx-colors';
  @import '../../scss-shared/nx-variables';
+ @import '../../scss-shared/nx-text-helpers';
+
+.nx-tabs {
+  margin: $nx-spacing-md 0 $nx-spacing-l 0;
+}
```

Comment:
I know you're not a fan of the separate files per component, but given that we have them, I think this should go into the NxTabs.scss file.  As it stands now, it's not even in the right directory.  Also, if a consumer were to, for some reason, use `NxTabs` without also using `NxTab`, as it stands now they wouldn't get this style (only styles for components that are actually being used get bundled in a properly configured build).

Reply:
> I know you're not a fan of the separate files per component, but given that we have them, I think this should go into the NxTabs.scss file. As it stands now, it's not even in the right directory. Also, if a consumer were to, for some reason, use `NxTabs` without also using `NxTab`, as it stands now they wouldn't get this style (only styles for components that are actually being used get bundled in a properly configured build).

I'm 100% in favor of a separate SCSS file for each component, I'm not very keen on cutting a single component up into four parts, that seems unnecessarily fussy. But yes this code was in the wrong place, fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/a24a17bb4ebe7479f1dbcc5823a98e7ddb316e22

Reply:
> I'm 100% in favor of a separate SCSS file for each component, I'm not very keen on cutting a single component up into four parts, that seems unnecessarily fussy. But yes this code was in the wrong place, fixed here: a24a17b

Right.  There's nothing stopping someone from writing multiple closely-related React components in the same tsx file, and then all their styles in a single corresponding scss file.  That's probably what we should try to do in this situation in the future.